packages/block-editor/src/components/block-draggable/test/helpers.native.js

```diff
@@ -108,11 +108,11 @@ export const initializeWithBlocksLayouts = async ( blocks ) => {
 /**
  * Fires long-press gesture event on a block.
  *
- * @param {import('react-test-renderer').ReactTestInstance} block                  Block test instance.
- * @param {string}                                          testID                 Id for querying the draggable trigger element.
- * @param {Object}                                          [options]              Configuration options for the gesture event.
- * @param {boolean}                                         [options.failed]       Determines if the gesture should fail.
- * @param {number}                                          [options.triggerIndex] In case there are multiple draggable triggers, this specifies the index to use.
+ * @param {HTMLElement} block                  Block test instance.
```

Comment:
The (original) spirit of these changes make sense to me — removing a [deprecated package](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#deprecated-react-test-renderer). 

However, given the package remains a peer dependency for `testing-library/react-native` [for the time being](https://github.com/callstack/react-native-testing-library/issues/1593) and that React Native, to my understanding, doesn't support/relate to `HTMLElement` — [for now](https://github.com/facebook/react-strict-dom/tree/main?tab=readme-ov-file), at least —  these changes feel a bit misleading. 

Not necessarily requesting a change or stating we must rollback the changes, merely sharing thoughts to see how others perceive this. 

cc/ @WordPress/native-mobile

Reply:
Hmm, looks like you're right. I can't recall where I was looking when changing this initially, but it doesn't seem to make sense. 

In any case, the original purpose of the PR failed, so I'm happy to revert, see https://github.com/WordPress/gutenberg/pull/61439.